### ISSUE TITLE
Chromebrew can't install git.

### ISSUE THREAD

**Issue by duuuno** [chromebrew/chromebrew#7229]
## Description
Hi. I'm unable to install chromebrew. Throws an error about tar when extracting git.
If you executed a specific command:
```shell
chronos@localhost / $ curl -Ls git.io/vddgY | bash
```

Output:
```
Welcome to Chromebrew!


Doing initial setup for install in /usr/local.
This may take a while if there are preexisting files in /usr/local...

Resetting ownership of /usr/local/bin
Resetting ownership of /usr/local/etc
Resetting ownership of /usr/local/lib
Resetting ownership of /usr/local/share
Resetting ownership of /usr/local/tmp

Downloading information for Bootstrap packages...

[1/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/musl_zstd.rb --> musl_zstd.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/musl_zstd.rb

[2/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/pixz.rb --> pixz.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/pixz.rb

[3/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ca_certificates.rb --> ca_certificates.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ca_certificates.rb

[4/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/git.rb --> git.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/git.rb

[5/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/gmp.rb --> gmp.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/gmp.rb

[6/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ncurses.rb --> ncurses.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ncurses.rb

[7/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/xxhash.rb --> xxhash.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/xxhash.rb

[8/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/lz4.rb --> lz4.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/lz4.rb

[9/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/popt.rb --> popt.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/popt.rb

[10/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/libyaml.rb --> libyaml.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/libyaml.rb

[11/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/openssl.rb --> openssl.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/openssl.rb

[12/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/zstd.rb --> zstd.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/zstd.rb

[13/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/gcc.rb --> gcc.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/gcc.rb

[14/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/rsync.rb --> rsync.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/rsync.rb

[15/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ruby.rb --> ruby.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ruby.rb
DL% UL%  Dled  Uled  Xfers  Live   Qd Total     Current  Left    Speed
--  --      0     0    15     0     0 --:--:-- --:--:-- --:--:--     0      

Downloading Bootstrap packages...

Downloading musl_zstd...
######################################################################## 100.0%
Verifying musl_zstd...
musl_zstd-1.5.2-2-chromeos-x86_64.tar.xz: OK
Extracting musl_zstd ...
Installing musl_zstd ...
Updating version number of musl_zstd in device.json...
Downloading pixz...
######################################################################## 100.0%
Verifying pixz...
pixz-1.0.7-0829-2-chromeos-x86_64.tar.xz: OK
Extracting pixz ...
Installing pixz ...
Updating version number of pixz in device.json...
Downloading ca_certificates...
######################################################################## 100.0%
Verifying ca_certificates...
ca_certificates-20210119-2-chromeos-x86_64.tar.xz: OK
Extracting ca_certificates ...
Installing ca_certificates ...
Updating version number of ca_certificates in device.json...
Downloading git...
######################################################################## 100.0%
Verifying git...
git-2.37.0-chromeos-x86_64.tar.zst: OK
Extracting git ...
tar: Child died with signal 11
tar: Error is not recoverable: exiting now
chronos@localhost / $ ^C

```

## Versions and system information
```***``` wasn't working sorry, text wouldn't be visible
```
chronos@localhost / $ cat /etc/lsb-release
CHROMEOS_ARC_ANDROID_SDK_VERSION=28
CHROMEOS_ARC_VERSION=8807643
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={9496CDE8-85E6-4118-960F-E26DC0C69FD6}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={9496CDE8-85E6-4118-960F-E26DC0C69FD6}
CHROMEOS_RELEASE_BOARD=grunt-signed-mp-v6keys
CHROMEOS_RELEASE_BRANCH_NUMBER=131
CHROMEOS_RELEASE_BUILDER_PATH=grunt-release/R103-14816.131.0
CHROMEOS_RELEASE_BUILD_NUMBER=14816
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=103
CHROMEOS_RELEASE_DESCRIPTION=14816.131.0 (Official Build) stable-channel grunt 
CHROMEOS_RELEASE_KEYSET=mp-v6
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_UNIBUILD=1
CHROMEOS_RELEASE_VERSION=14816.131.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=14816.131.0
chronos@localhost / $
```
***
***

**Comment by duuuno** [comment]
Not sure how, but, um, git extracts now. Another program, popt, still cannot though :(
```Welcome to crosh, the Chrome OS developer shell.

If you got here by mistake, don't panic!  Just close this tab and carry on.

Type 'help' for a list of commands.

If you want to customize the look/behavior, you can use the options page.
Load it by using the Ctrl-Shift-P keyboard shortcut.

crosh> shell
chronos@localhost / $ curl -Ls git.io/vddgY | bash
Welcome to Chromebrew!
Please enter the developer mode password
Password: 


Doing initial setup for install in /usr/local.
This may take a while if there are preexisting files in /usr/local...

Resetting ownership of /usr/local/bin
Resetting ownership of /usr/local/etc
Resetting ownership of /usr/local/lib
Resetting ownership of /usr/local/share
Resetting ownership of /usr/local/tmp

Downloading information for Bootstrap packages...

[1/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/musl_zstd.rb --> musl_zstd.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/musl_zstd.rb

[2/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/pixz.rb --> pixz.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/pixz.rb

[3/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ca_certificates.rb --> ca_certificates.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ca_certificates.rb

[4/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/git.rb --> git.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/git.rb

[5/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/gmp.rb --> gmp.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/gmp.rb

[6/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ncurses.rb --> ncurses.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ncurses.rb

[7/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/xxhash.rb --> xxhash.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/xxhash.rb

[8/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/lz4.rb --> lz4.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/lz4.rb

[9/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/popt.rb --> popt.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/popt.rb

[10/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/libyaml.rb --> libyaml.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/libyaml.rb

[11/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/openssl.rb --> openssl.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/openssl.rb

[12/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/zstd.rb --> zstd.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/zstd.rb

[13/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/gcc.rb --> gcc.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/gcc.rb

[14/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/rsync.rb --> rsync.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/rsync.rb

[15/15]: https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ruby.rb --> ruby.rb
--_curl_--https://raw.githubusercontent.com/chromebrew/chromebrew/master/packages/ruby.rb
DL% UL%  Dled  Uled  Xfers  Live   Qd Total     Current  Left    Speed
--  --      0     0    15     0     0 --:--:-- --:--:-- --:--:--     0      

Downloading Bootstrap packages...

Downloading musl_zstd...
######################################################################## 100.0%
Verifying musl_zstd...
musl_zstd-1.5.2-2-chromeos-x86_64.tar.xz: OK
Extracting musl_zstd ...
Installing musl_zstd ...
Updating version number of musl_zstd in device.json...
Downloading pixz...
######################################################################## 100.0%
Verifying pixz...
pixz-1.0.7-0829-2-chromeos-x86_64.tar.xz: OK
Extracting pixz ...
Installing pixz ...
Updating version number of pixz in device.json...
Downloading ca_certificates...
######################################################################## 100.0%
Verifying ca_certificates...
ca_certificates-20210119-2-chromeos-x86_64.tar.xz: OK
Extracting ca_certificates ...
Installing ca_certificates ...
Updating version number of ca_certificates in device.json...
Downloading git...
######################################################################## 100.0%
Verifying git...
git-2.37.0-chromeos-x86_64.tar.zst: OK
Extracting git ...
Installing git ...
Adding new information on git to device.json...
Downloading gmp...
######################################################################## 100.0%
Verifying gmp...
gmp-6.2.1-chromeos-x86_64.tar.xz: OK
Extracting gmp ...
Installing gmp ...
Adding new information on gmp to device.json...
Downloading ncurses...
######################################################################## 100.0%
Verifying ncurses...
ncurses-6.3-20211106-chromeos-x86_64.tpxz: OK
Extracting ncurses ...
Installing ncurses ...
Adding new information on ncurses to device.json...
Downloading xxhash...
######################################################################## 100.0%
Verifying xxhash...
xxhash-0.8.0-1-chromeos-x86_64.tar.xz: OK
Extracting xxhash ...
Installing xxhash ...
Adding new information on xxhash to device.json...
Downloading lz4...
######################################################################## 100.0%
Verifying lz4...
lz4-1.9.2-chromeos-x86_64.tar.xz: OK
Extracting lz4 ...
Installing lz4 ...
Adding new information on lz4 to device.json...
Downloading popt...
######################################################################## 100.0%
Verifying popt...
popt-1.18-7182e46-1-chromeos-x86_64.tar.zst: OK
Extracting popt ...
tar: Child died with signal 11
tar: Error is not recoverable: exiting now
chronos@localhost / $ ```

**Comment by duuuno** [comment]
Also, running `git` leads to a segmentation fault.
```
chronos@localhost / $ git
Segmentation fault (core dumped)
chronos@localhost / $ ```

**Comment by satmandu** [comment]
Does this machine have an AMD CPU?

**Comment by duuuno** [comment]
> Does this machine have an AMD CPU?

Yes.

**Comment by satmandu** [comment]
There's currently an issue we're seeing with installs on AMD CPUs and the musl static binaries we are using, such as git.

**Comment by satmandu** [comment]
Could you try this just to see if it fixes git?

```
curl -OLf https://github.com/chromebrew/chromebrew/raw/master/install.sh
chmod +x install.sh
OWNER=satmandu BRANCH=git2371 ./install.sh

```

**Comment by satmandu** [comment]
And also what does this give you?

```
zstd --version
/usr/local/bin/zstd --version
```

**Comment by Zopolis4** [comment]
Should be fixed by #7238 now.
